overlord/servicestate/quota_control.go

```diff
@@ -68,6 +68,7 @@ func quotaGroupsAvailable(st *state.State) error {
 		return fmt.Errorf("cannot use quotas with incompatible systemd: %v", systemdVersionError)
 	}
 
+	// TODO: remove this check
```

Comment:
Will be removed in https://github.com/snapcore/snapd/pull/11991

Reply:
you could also use featuresEnabledWhenUnset to switch things first and remove the check later

Reply:
Ah nice suggestion, did not know about the `featuresEnabledWhenUnset` setup.

Reply:
But this would coincide with doing a quota-group OR journal-quota in the second review comment. This would make journal-quota enabled by default. So maybe keep it like it is?

Reply:
yes, I mentioned in my other comment that now we need to leave as is